.travis.yml

```diff
@@ -23,15 +21,23 @@ matrix:
   - os: osx
     compiler: gcc
 
+addons:
```

Comment:
Is there a way to use a similar syntax for brew like as here?

Reply:
And how do we do this when we need extra ppa?

Reply:
According to travis-ci/travis-ci#5377 there is nothing like this for homebrew yet, unfortunately. For apt, you can ppas to a `sources` list (copied from travises documentation):
```yaml
addons:
  apt:
    sources:
    - ubuntu-toolchain-r-test
    packages:
    - gcc-4.8
    - g++-4.8
```

Reply:
I've had a look into it. We can only use white-listed sources.
In my sub-branch I'll need globjects and glbinding, which are not whitelisted. That means, that we need sudo requiered either way.

Reply:
> At this moment, we are unable to continue processing the APT sources requests. If you require a new source, please use `sudo: required` in your `.travis.yml`  file.


See https://github.com/travis-ci/apt-source-whitelist

Reply:
I thought glbinding and globjects had official ubuntu/debian packages, or are they too outdated? Anyway, that's unfortunate, but the linux builds will probably still be faster than the macOS builds.

I guess we just revert to manual apt installation when merging your PR? You're changing the `.travis.yml` substantially anyway.

Reply:
Ok, fine by me :)